travis/tests/static/model/configuration/static.fac

```diff
@@ -0,0 +1,187 @@
+********************************************************************* ;
+* A citation to the MCM website and the relevant mechanism          * ;
+* construction protocols should be given in any publication using   * ;
+* information obtained from this source, using the following or     * ;
+* comparable wording:                                               * ;
+* The chemical mechanistic information was taken from the Master    * ;
+* Chemical Mechanism, MCM v3.3.1 (ref), via website:                  * ;
+* http://mcm.leeds.ac.uk/MCM.                                       * ;
+* The reference should be: (Jenkin et al., Atmos. Environ., 31, 81, * ;
+* 1997; Saunders et al., Atmos. Chem. Phys., 3, 161, 2003), for     * ;
+* non aromatic schemes; (Jenkin et al., Atmos. Chem. Phys., 3,  * ;
+* 181, 2003; Bloss et al., Atmos. Chem. Phys., 5, 641, 2005), for   * ;
+* aromatic schemes; (Jenkin et al., Atmos. Chem. Phys.,  12, * ;
+* 5275, 2012), for the beta-caryophyllene scheme and (Jenkin et al., ;
+* Atmos. Chem. Phys., 15, 11433, 2015), for the isoprene scheme.  * ;
+********************************************************************* ;
+* MCMv3.3.1 Subset generated for the following species:   ;
+* CH3OH C2H5OH BUT2OL ;
+*;
+* Variable definitions.  All species are listed here.;
+*;
+VARIABLE
+ O O3 ;
+****************************************************** ;
+*;
+* Generic Rate Coefficients ;
+*;
+KRO2NO = 2.7D-12*EXP(360/TEMP) ;
```

Comment:
A minor point. It may be better to use the minimal mechanism (https://github.com/AtChem/AtChem2/wiki/2.1-Chemical-Mechanism#minimal-mechanism-file) for these tests, if only for clarity. and substitute the standard MCM header with a comment of that the test does.

Reply:
Yes, that might be cleaner.